### PR TITLE
Added section on translation friendly writing

### PR DESCRIPTION
**Pull request by dariavladykina**
The "Writing for the Web" chapter has been updated with a new section authored by Julia, "Writing for a global audience".

### CODE REVIEW

**Review by janajaeger — APPROVED** [review]
This looks very nice! Thanks a million!

**Review by jfaltenbacher — APPROVED** [review]
Thank you Daria

File: xml/docu_styleguide.webwriting.xml [hunk 1]
```diff
@@ -159,4 +159,89 @@
...
+   <listitem>
+   <para>
+   Write for the world (if possible). Do not use country-specific words and examples.
+   Use common international examples instead.
+   </para>
+   </listitem>
+   <listitem>
+   <para>
+   Use only as many graphics as needed. As each graphic or screenshot needs
+   to be localized as well, keep it to the minimun.
```
Comment (dariavladykina) [inline xml/docu_styleguide.webwriting.xml:217]:
```suggestion
   to be localized as well, keep it to the minimum.
```

**Review by dariavladykina — COMMENTED** [review]
correction

**Pull request merged**